### ISSUE TITLE
Test / fix OnlyStaticPeers

### ISSUE THREAD

**Issue by tkstanczak** [NethermindEth/nethermind#1939]

**Comment by dceleda** [comment]
@tkstanczak @LukaszRozmej 
Any description of what needs to be done here?

**Comment by kamilchodola** [comment]
I feel like I know exactly what is happening here.

Setting static peer is pretty painful (not only in Nethermind but also in other clients).

Setting this in ELs can be done by disabling discovery and add peer by jsonRPC call (admin_addPeer)

In Nethermind with that flag it feels like we should be capable to do it like this also:
1. Set above mentioned flag to true
2. Add peer by admin_addPeer or, by Network.StaticPeers or by static-peers.json file

Problem is that it is not enough and not only static peer is added but also other peers are added as usual causing this isolation test not to work properly.

Workaround (at least for me) to make it work automatically on startup is to use this pretty huge set of params:
```
Network.MaxActivePeers=0 
Network.OnlyStaticPeers=true Network.StaticPeers=enode://a152c528aa749e73913d14fbbc289a369ee69c2f166efda066ebbde4d449fead55ab94efc0a356887fe92b02eab293a8e3be0dae9de9a3063374a248d591963f@172.234.229.46:30303 
Init.DiscoveryEnabled=false 
Network.DiscoveryDns=dummy 
Sync.SnapSync=true 
Network.DiagTracerEnabled=true
```

where it should work simply with:
```
Network.OnlyStaticPeers=true
Network.StaticPeers=enode://a152c528aa749e73913d14fbbc289a369ee69c2f166efda066ebbde4d449fead55ab94efc0a356887fe92b02eab293a8e3be0dae9de9a3063374a248d591963f@172.234.229.46:30303
```

Will add that to stability dashboard 

cc @MarekM25

**Comment by LukaszRozmej** [comment]
I think the original problem was that with `OnlyStaticPeers` we had high CPU usage.

**Comment by kamilchodola** [comment]
This was fixed by @asdacap in #7514
